ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -13,7 +13,11 @@ if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
   exit 1
 fi
 
-echo "Installing from release ${RELEASE_IMAGE_LATEST}"
+if [[ "${OPENSHIFT_UPGRADE:-false}" == "false" ]]; then
```

Comment:
Why do we need this here? I'd rather just remove it and have us print this during the install step, no need to have `conf` care if it does nothing with this value

Reply:
Moved the echo to the install step.